lib/utils.ts

```diff
@@ -516,6 +516,10 @@ export function asSafeVer(semver: string | number | null | undefined) {
                 return validated;
             }
         }
+
+        if (semver.includes('trunk') || semver.includes('main')) {
+            return '99999999.99999.999';
```

Comment:
bit of a hack but I hve no better idea :) Mayyybe we should make a `const` value for the `99999.99.99` constant though?

Reply:
Ohhhh! scratch that! I didn't sot they were actualy different...

So maybe...name them like you've done in the tests? Then old eyes like mine might realise they're different and it's on purpose :)

Reply:
yeah that's what i was thinking as well

Reply:
fixed